jupyter/d2l-java/chapter_convolutional-modern/alexnet.ipynb

```diff
@@ -0,0 +1,688 @@
+   "outputs": [],
+   "source": [
+    "%load ../utils/plot-utils.ipynb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ai.djl.Model;\n",
+    "import ai.djl.basicdataset.FashionMnist;\n",
+    "import ai.djl.metric.Metrics;\n",
+    "import ai.djl.ndarray.NDArray;\n",
+    "import ai.djl.ndarray.NDManager;\n",
+    "import ai.djl.ndarray.types.DataType;\n",
+    "import ai.djl.ndarray.types.Shape;\n",
+    "import ai.djl.nn.Activation;\n",
+    "import ai.djl.nn.Blocks;\n",
+    "import ai.djl.nn.SequentialBlock;\n",
+    "import ai.djl.nn.convolutional.Conv2D;\n",
+    "import ai.djl.nn.core.Linear;\n",
+    "import ai.djl.nn.pooling.Pool;\n",
+    "import ai.djl.training.DefaultTrainingConfig;\n",
+    "import ai.djl.training.EasyTrain;\n",
+    "import ai.djl.training.Trainer;\n",
+    "import ai.djl.training.dataset.Dataset;\n",
+    "import ai.djl.training.dataset.ArrayDataset;\n",
+    "import ai.djl.training.evaluator.Accuracy;\n",
+    "import ai.djl.training.listener.TrainingListener;\n",
+    "import ai.djl.training.loss.Loss;\n",
+    "import ai.djl.training.optimizer.Optimizer;\n",
+    "import ai.djl.training.optimizer.learningrate.LearningRateTracker;\n",
+    "import tech.tablesaw.api.*;\n",
+    "import tech.tablesaw.plotly.api.*;\n",
+    "import tech.tablesaw.plotly.components.*;\n",
+    "import tech.tablesaw.plotly.Plot;\n",
+    "import tech.tablesaw.plotly.components.Figure;\n",
+    "import org.apache.commons.lang3.ArrayUtils;"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "attributes": {
+     "classes": [],
+     "id": "",
+     "n": "1"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "NDManager manager = NDManager.newBaseManager();\n",
+    "\n",
+    "SequentialBlock block = new SequentialBlock();\n",
+    "\n",
+    "block\n",
+    "                .add(Conv2D.builder()\n",
+    "                        .setKernelSize(new Shape(11, 11))\n",
+    "                        .optStride(new Shape(4, 4))\n",
+    "                        .optBias(false)\n",
+    "                        .setFilters(96).build())\n",
+    "                .add(Activation::relu)\n",
+    "                .add(Pool.maxPool2DBlock(new Shape(3, 3), new Shape(2, 2)))\n",
+    "                .add(Conv2D.builder()\n",
+    "                        .setKernelSize(new Shape(5, 5))\n",
+    "                        .optPadding(new Shape(2, 2))\n",
+    "                        .optBias(false)\n",
+    "                        .setFilters(256).build())\n",
+    "                .add(Activation::relu)\n",
+    "                .add(Pool.maxPool2DBlock(new Shape(3, 3), new Shape(2, 2)))\n",
+    "                .add(Conv2D.builder()\n",
+    "                        .setKernelSize(new Shape(3, 3))\n",
+    "                        .optPadding(new Shape(1, 1))\n",
+    "                        .optBias(false)\n",
```

Comment:
Please double check with Gluon's default settings. Bias are required by default for both DJL and Gluon